docs/cluster-deployment/stripe.rst

```diff
@@ -135,19 +136,42 @@ Provide billing information
 ===========================
 
 Finally, you will be taken to a new screen where you can fill out your billing
-information. Our payment processing is supported by `Stripe`_. At the bottom
-right you can find the cards accepted by Crate.io. When you have filled out the
-necessary information, click *Deploy* below it to deploy your cluster. Do not
-forget to accept financial authorization by ticking the box at the bottom.
+information. Our credit card payment processing is supported by `Stripe`_. If
+you have an existing AWS or Azure subscription, you can also pay using that.
+You can choose the payment method right after you configure your cluster and
+click **Next**.
+
+.. image:: ../_assets/img/payment-method.png
+   :alt: Payment method screen
+
+Credit card
+-----------
+
+You can find the cards accepted by Crate.io at the bottom right part of the screen.
+When you have filled out the necessary information, click *Deploy* below it to
+deploy your cluster. Do not forget to accept financial authorization by ticking
+the box at the bottom.
 
 .. image:: ../_assets/img/stripe-billing.png
    :alt: Billing information screen
 
+AWS/Azure
+---------
+
+The other payment option is to use AWS or Azure marketplace subscription.
+Simply choose the one you'd like to use and you will be brought to
+the respective marketplace page where you can confirm the subscription.
+
+.. image:: ../_assets/img/payment-method-marketplaces.png
+   :alt: Billing information screen
+
+---
+
 The payment and billing information you have submitted will be saved in the
 Billing tab of the Organization overview screen in the CrateDB Cloud Console
 (i.e., the fifth tab from the left on the same screen you arrived at).
 
-You can at any time check your current accumulated bill at the bottom left of
+You can, at any time, check your current accumulated bill at the bottom left of
```

Comment:
To resolve any grammatical ambiguity here you could say something like

```suggestion
Your current accumulated bill is shown in the billing meter in the bottom left of the CrateDB Cloud Console screen:
```

or also if you prefer the original sentence you could say something like

```
You can check your current accumulated bill at any time by viewing the billing meter in the bottom left of the CrateDB Cloud Console screen
```